biz.aQute.bnd.exporters/.gitignore

```diff
@@ -1,3 +1,6 @@
 /bin/
 /bin_test/
 /generated/
+/test-ws/**/*.esa
```

Comment:
It is better for each test to copy the test workspace into `generated/tmp/test/<testmethodname>` for each test and use that workspace. This was we avoid writing to the src folder and needing git ignore. We also avoid issues with locked files on Windows when multiple test methods share the same workspace. This can be done in the @Before method of the test case.